### ISSUE TITLE
style: Edge seems not supporting css shapes...

### ISSUE THREAD

**Issue by cousquer** [cousquer/CardWebComponents#40]
**Describe the bug**
Edge seems not supporting css shapes...
**label:** _trivial_

**To Reproduce**
open in edge [https://cousquer.github.io/CardWebComponents/](https://cousquer.github.io/CardWebComponents/)

![edge](https://user-images.githubusercontent.com/1481430/49672353-18d72e00-fa6b-11e8-91ae-d6875488c96a.png)

**Expected behavior**
![in chrome](https://user-images.githubusercontent.com/1481430/49672396-36a49300-fa6b-11e8-92a2-70311045f4d8.png)

**Remediation**
 - give a try on vendor prefix
 - if it doesn't fix the problem small svg only for ie/edge... 😞 

**Desktop (please complete the following information):**
 - OS: [windows 10]
 - Browser [edge]
 - Version [all]

**Additional context**

**Comment by cousquer** [comment]
this can work for edge:
```
SVG clip-path for SVG elements
Specification 

<clipPath id="clipping">
  <polygon points="98.4999978 153.75..."/>
</clipPath>
.item {
  clip-path: url(#clipping);
  }
```

**Comment by cousquer** [comment]
```
/* Microsoft Edge Browser 12+ (All) - @supports method */

@supports (-ms-ime-align:auto) {
  .selector { property:value; } 
}
```

**Comment by cousquer** [comment]
resolved in #41
